NEWS.rst

```diff
@@ -9,6 +9,34 @@
 
 .. towncrier release notes start
 
+24.1b2 (2024-06-11)
+===================
+
+Features
+--------
+
```

Comment:
I mislabeled this news fragment when I dropped colorama :upside_down_face:. This should be moved down. Sorry for the extra busy work!